### DISCUSSION TITLE
Help with install

### DISCUSSION THREAD

**Discussion by brendanjones** [filips123/PWAsForFirefox#126] [Q&A] [answered]
Can't figure out what I'm doing wrong/need to do. Any help appreciated, I am not a dev!

I'm on MacOS, so the install steps are:

1. Make sure [Homebrew](https://brew.sh/) is installed.
2. Install the formula using brew: brew install firefoxpwa
**3. Follow instructions displayed in the caveats section of installation output to link the manifest.**
4. After the formula is installed and the manifest is linked, you should be automatically proceeded to the next step. If nothing changes after around 30 seconds, restart the browser.

I'm stuck on number 3. I've no idea what this caveats section is. It looks like firefoxpwa is installed, because if I try it again I get:

```
~ % brew install firefoxpwa
Running `brew update --preinstall`...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
ffmpeg@4
==> Updated Formulae
Updated 46 formulae.
==> Updated Casks
Updated 11 casks.

Warning: firefoxpwa 1.4.0 is already installed and up-to-date.
```

Have restarted browser with no effect. Any ideas?

**Comment by filips123** [comment, marked as answer]
I'm not sure why this didn't happen, but installing `firefoxpwa` using brew should display some additional instructions. You can view them by running `brew info firefoxpwa`. It will display something like "Caveats: To use the browser extension, manually link the app manifest with" and you need to run commands displayed there.

**Reply by brendanjones** [reply]
Brilliant, that indeed pulled up the required info. All installed, thanks for the support!
